src/braket/circuits/circuit.py

```diff
@@ -115,7 +117,11 @@ def method_from_subroutine(self, *args, **kwargs) -> SubroutineReturn:
         setattr(cls, function_name, method_from_subroutine)
 
         function_attr = getattr(cls, function_name)
-        setattr(function_attr, "__doc__", func.__doc__)
+        # docstrings don't like a single newline for some reason.
+        function_docstring = (
+            "\n\n".join([cls_docstring, func.__doc__]) if len(cls_docstring) > 0 else func.__doc__
+        )
+        setattr(function_attr, "__doc__", function_docstring)
```

Comment:
Do you whether setting `__doc__` in this way allows the docstring to appear in IDEs such as VS Code?

Reply:
I don't have VSCode setup so I am unsure at the moment.